public/css/styles.css

```diff
@@ -0,0 +1,103 @@
+/* The Modal (background) */
+
+.modal {
+    display: none;
+    /* Hidden by default */
+    position: fixed;
+    /* Stay in place */
+    z-index: 1;
+    /* Sit on top */
+    padding-top: 75px;
+    /* Location of the box */
+    left: 0;
+    top: 0;
+    width: 100%;
+    /* Full width */
+    height: 100%;
+    /* Full height */
+    overflow: auto;
+    /* Enable scroll if needed */
+    background-color: rgb(0, 0, 0);
+    /* Fallback color */
+    background-color: rgba(0, 0, 0, 0.9);
+    /* Black w/ opacity */
+    overflow-y: hidden;
```

Comment:
the comments confused me at first :) I think it’s more common to put comments on top of the line that they describe, rather than below. Do you usually do it the other way around?

Reply:
No that's not it. :joy: 
I was testing out several image modals and it's from the sample which I liked, in fact this is the first time I'm seeing comments below code too. :laughing: